layouts/partials/simpleicons.html

```diff
@@ -0,0 +1,6 @@
+<span class="inline-svg" >
+{{- $fname:=print "simpleicons/" . ".svg" -}}
+{{- $path:="<path" -}}
+{{- $fill:="<path fill=\"currentColor\"" -}}
+{{ replace (readFile $fname) $path $fill | safeHTML }}
+</span>
```

Comment:
This feels like a complicated way of including an svg on a page to me? Am I missing something?